astro.sidebar.ts

```diff
@@ -138,6 +138,7 @@ export const sidebar = [
 					'reference/experimental-flags/serialized-configuration',
 					'reference/experimental-flags/preserve-scripts-order',
 					'reference/experimental-flags/heading-id-compat',
+					'reference/experimental-flags/fonts',
```

Comment:
NOTE: I am LEAVING THIS HERE ON PURPOSE so it will be easier to merge with the other PRs into the 5.7.0 branch. After that, I'll move it up higher in the list.